command/inspect.go

```diff
@@ -52,6 +50,50 @@ func (c *InspectCommand) Run(args []string) int {
 		return 1
 	}
 
+	// If args not specified but output format is specified, format and output the jobs data list
```

Comment:
I think you can clean this one up by removing the -json option since that is true by default.

Further, since the formatter takes `interface{}`, I think we can cleanup the code by doing something like:

```
var toFormat interface{}
if len(args) == 1 {
  toFormat := getJob()
} else {
  toFormat = listJobs()
}

formatter.Transform(toFormat)
```